### ISSUE TITLE
Valet Linux sock Permission denied

### ISSUE THREAD

**Issue by axecore** [cpriego/valet-linux#253]
please help
![vungul](https://user-images.githubusercontent.com/26722043/70054893-544dff00-15d0-11ea-9272-c48dc3bc9a38.png)

**Comment by cpouldev** [comment]
Same issue as https://github.com/cpriego/valet-linux/issues/251

You updated your php to 7.4. That broke valet

**Comment by rgb-one** [comment]
You need to revert the other php packages you have installed as well.

**Comment by jmarcher** [comment]
I did not had the time to test Valet with php 7.4 for the time being please stick with PHP 7.3.

**Comment by cesasol** [comment]
This update on archlinux indeed broke valet, but I managed to get it up by modifying the systemd unit as follows:

```
- ProtectHome=true
+ ProtectHome=false
...
- CapabilityBoundingSet=CAP_SETGID CAP_SETUID
+ # CapabilityBoundingSet=CAP_SETGID CAP_SETUID
```

Those two settings are protecting the user and ther home folder from php-fpm, so disabling it allows valet to work as intended.

**Comment by simtrax** [comment]
> CapabilityBoundingSet

Works. But looking forward to a solution to this problem.

**Comment by muhammetsait** [comment]
If we change the location of `valet.sock` to somewhere else then `CapabilityBoundingSet` is no longer a problem but we would still need to override `ProtectHome`

Here's my modified `/etc/php/php-fpm.d/valet.conf`

```
- listen = VALET_HOME_PATH/valet.sock
+ listen = /run/php-fpm/valet.sock

- listen.owner = VALET_USER
- listen.group = VALET_GROUP
+ listen.acl_users = VALET_USER
+ listen.acl_groups = VALET_GROUP
```

Also, `/etc/nginx/sites-available/valet.conf`
```
- fastcgi_pass unix:VALET_HOME_PATH/valet.sock;
+ fastcgi_pass unix:/run/php-fpm/valet.sock;
```

**Comment by mackerson** [comment]
I'm also affected by this bug - what's the most direct way to get the recently patched code? Doesn't look like there was a version bump alongside of it to address the hotfix so composer global update isn't pulling it. Uninstall / reinstall?

**Comment by jmarcher** [comment]
This should be fixed on v2.2.9

**Comment by pschilly** [comment]
I am running v2.2.9 now - and use the command:   valet use 7.4 to enable 7.4 but it is still using 7.2...

Yes I do have php-7.4-fpm installed as well



Suggestions?
